vllm/entrypoints/openai/api_server.py

```diff
@@ -640,6 +691,10 @@ async def fake_stream_generator() -> AsyncGenerator[str, None]:
     else:
         served_model = args.model
 
+    disabled_endpoints = args.disable_endpoints
```

Comment:
This seems like a separate features no?

Reply:
Yes at this point since I couldn't implement a hard-fail on loading templates at startup easily. I mentioned it in the other PR last night that I left it in place because I had already implemented it at Simon's request. I can tear that out if necessary.

Reply:
Removed.